File: cpp/Queues/Queues.cpp
#include<iostream>
using namespace std;
class queues {
   int max;
    int arr[max];
    int top = -1;
    int start = 0;

  public:   queues(){
       cin >> max;
     }
     int push(){
        int value;
       if (top ==max-1) {
         cout << "overflow" << endl;
       }
       else{
          cout << "Enter A Number";
          cin >> value;
          arr[++top] = value;

       }
     }
     int pop(){
       if (top == -1) {
         cout << "undeFlow";

       }
       else{
         start++;
       }

     }
     int show(){
       for ( int i=start; i<max  i++) {
         cout << arr[i];
       }
     }


};
int main(int argc, char const *argv[]) {
  queues acess;
  int option;
  while (true) {
    /* code */

cout << "1 \t push" << '\n';
cout << "2 \t pop" << '\n';
cout << "3 \t show" << '\n';
cout << "4 \t Exit" << '\n';
cin >> option;
switch (option) {
  case 1:
  acess.push();
  break;
  case 2:
  acess.pop();
  break;
  case 3:
  acess.show();
  break;
  case 4:
   exit(0);
   break;
}


}
  return 0;
}

File: Java/Queues/Queues.java
import java.util.*;
import java.util.ArrayList;
import java.util.Scanner;
 public class Queues{

    int start ,top ,max;
    Scanner Takeinput;
    ArrayList<Integer> q;
    public Queues(){
      start = 0;
      top = -1;
     q = new ArrayList<>();
     Takeinput = new Scanner(System.in);
    System.out.println("what ll size of Array");
    max = Takeinput.nextInt();

    }
    void push(){
      int value;
       if(top ==max-1) {
          System.out.println("overflow");
       }
       else{
        value = Takeinput.nextInt();
         q.add(value);
         top++;
       }
    }
    void pop(){
      if(top == -1) {
        System.out.println("undeflow");
      }
      else{
         q.remove(start);

      }
    }
    void show(){
      if(top == -1) {
        System.out.println("Ematy");
      }
      else{
        for(Integer df:q){
          System.out.println(df);
        }
      }
    }
public static void main(String...ar){
int option;
Queues object = new Queues();
 while(true){
  System.out.println("1 \t push");
  System.out.println("2 \t pop");
  System.out.println("3 \t show");
  System.out.println("4 \t Exit");
  option = object.Takeinput.nextInt();
 switch (option) {
   case 1:
   object.push();
   break;
   case 2:
   object.pop();
   break;
   case 3:
   object.show();
   break;
   case 4:
   System.exit(0);
   break;
 }
 }
}
 }
